.changeset/olive-zebras-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/internet-header': minor
+---
+
+Enables nesting the header in a different scroll container than the `<body>` element. The header stickyness and logo animation logic will automatically attach to the nearest scrolling container instead of the document when nested in a container that has `overflow: auto | scroll` set.
```

Comment:
```suggestion
Enabled nesting the header in a different scroll container than the `<body>` element. The header stickyness and logo animation logic will automatically attach to the nearest scrolling container instead of the document when nested in a container that has `overflow: auto | scroll` set.
```